test/contrib/automotive/uds_utils.uts

```diff
@@ -9,9 +9,9 @@ import pickle
 import scapy.modules.six as six
 
 if six.PY3:
-    exec(open("test/contrib/automotive/interface_mockup.py").read())
+    exec(open("contrib/automotive/interface_mockup.py").read())
```

Comment:
Please call the test always from the scapy root directory
```
sudo ./test/run_tests -2 -t test/contrib/automotive/uds_utils.uts -F
```
Otherwise the tests will fail on Scapys CI system.

Reply:
Thx.